utils/misc/misc.go

```diff
@@ -276,13 +277,30 @@ func UnZipSingleFile(outputfile string, filename string) {
 	rc.Close()
 }
 
-func RemoveFilePaths(filepaths ...string) {
-	for _, filepath := range filepaths {
+// RemoveFilePaths removes filePaths as well as cleans up the empty folder structure.
+func RemoveFilePaths(filePaths ...string) {
```

Comment:
Why do we need filepaths to be a `...string`. It looks like all the callers of this send only one path.

Reply:
RemoveFilePaths is an older implementation. I just updated the comment at the top and added cleanup for folders here.
Since RemoveFilePaths is being used at other places as well like batchrouter.go, azure-synapse, clickhouse.go, identify.go where we are using ...